src/java.net.http/share/classes/jdk/internal/net/http/common/Utils.java

```diff
@@ -617,23 +617,27 @@ public static int getIntegerProperty(String name, int defaultValue) {
     }
 
     public static SSLParameters copySSLParameters(SSLParameters p) {
-        SSLParameters p1 = new SSLParameters();
+        final SSLParameters p1 = new SSLParameters();
```

Comment:
nit: I'm not a fan of labeling local variables as final; it doesn't add any value as far as I can tell. Can we keep it non-final?

Reply:
Hello Daniel, I use it as a habit to mark even local variables as final to prevent accidental reassignment, plus while reading the code to make it easier to remember that the variable won't get reassigned after that line.

But that's just a personal preference. Plus this line isn't new and the rest of the code doesn't use `final`, so I have updated the PR to remove this `final` that was introduced in the previous revision.